include/caf/power_manager.rst

```diff
@@ -0,0 +1,152 @@
+.. _caf_power_manager:
```

Comment:
We will have to align documentation with other CAF modules and rephrase notes that are specific to nRF Desktop.
I think we should do it in separate PR (this one already introduces quite a lot of changes).

Reply:
Witch notes still references to nrf_desktop? I have done some cleanup... maybe I missed something.

Reply:
`If the device is not connected through USB, the module counts time elapsed since the last user interaction (that is, since the last HID report sent from the device).`
We should mention `keep_alive_event` here instead